examples/neff_shuffle_test.go

```diff
@@ -0,0 +1,106 @@
+package examples
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	kproof "go.dedis.ch/kyber/v3/proof"
+	"go.dedis.ch/kyber/v3/shuffle"
+)
+
+/*
+This example illustrates how to use the Neff shuffle protocol with simple,
+single pairs.
+*/
+func Test_Example_Neff_Shuffle_Simple(t *testing.T) {
+	numPairs := 3
+
+	// generate random pairs
+
```

Comment:
Uniformity of comments: don't have an empty line between the comment an the code.